templates/pages/person.html

```diff
@@ -1,65 +1,44 @@
 {{> global/global-header}}
 {{> global/search-main reduced="reduced"}}
 
-<div class="page" role="main">
+{{#if error}}
+  {{> global/error}}
+{{else}}
```

Comment:
This doesn't need to be here, and complicates the template - let the error plugin handle 404 errors and display a 404 page.